apps/web/src/views/Swap/hooks/useTradingRewardTokenList.tsx

```diff
@@ -9,15 +9,15 @@ const useTradingRewardTokenList = () => {
   const { data } = useAllTradingRewardPair(RewardStatus.ALL)
 
   const uniqueAddressList = useMemo(() => {
-    const currentTime = new Date().getTime() / 1000
+    const currentTime = Date.now() / 1000
 
     // eslint-disable-next-line array-callback-return, consistent-return
     const activeRewardCampaignId = data.campaignIds.filter((campaignId) => {
       const incentive = data.campaignIdsIncentive.find((i) => i.campaignId === campaignId)
       if (incentive.campaignClaimTime > currentTime) {
         return campaignId
       }
-      return campaignId
+      return []
```

Comment:
Sorry I don't see the point of the filter function here. No matter we're returning `campaignId` or `[]`, I assume they're both truthy value? Which means the we actually don't filter anything at all?

Reply:
my bad should filter when `incentive.campaignClaimTime > currentTime`